person.rb

```diff
@@ -1,14 +1,15 @@
 require './nameable'
 
 class Person < Nameable
-  attr_reader :id
+  attr_reader :id, :rentals
   attr_accessor :name, :age
 
   def initialize(age, name = 'Unknown', parent_permission: true)
     super()
     @name = name
     @parent_permission = parent_permission
     @age = age
+    @rentals = []
   end
 
   def can_use_services?
```

Comment:
- You have not implemented the `has-many` relationship between the `person` class and the `rental` class. Please implement a method `add_rental` for this purpose 🙏.

Reply:
Hi @mckent05 , tks for the feedback, however, I don't see a good reason to add the add_rental to the Person class as a rental also requires a book and we don't necessarily need the Person class to know about the Book class, instead a Person solely needs to know about the Rental class - it would be cleaner to solely create rentals from the Rental class and not create rentals from the Person class as either 1. this would require the Person class to know about the Book class or 2. we would create rentals with no books. Let me know what you think - tks!